src/components/ChatEntry.js

```diff
@@ -1,22 +1,43 @@
 import React from 'react';
 import './ChatEntry.css';
+import TimeStamp from './TimeStamp.js';
 import PropTypes from 'prop-types';
 
 const ChatEntry = (props) => {
+  const timeEntry=props.timeStamp
+
+  const onLikeButtonClick = () => {
+    const updatedChat = {
+        id: props.id,
+        sender: props.sender,
+        body: props.body,
+        timeStamp: props.timeStamp,
+        liked: !props.liked
+    };
+
+    props.onUpdateChat(updatedChat);
+  };
```

Comment:
When you just pass in an id to props.onUpdateChat(), then this method can be refactored to just be one line: 
```javascript
const onLikeButtonClick = () => {
    props.onUpdateChat(props.id);
};